contents/careers/marketer.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Head of Marketing
+sidebar: Careers
+showTitle: true
+---
+## PostHog exists to increase the number of successful products in the world.
+
+While we have hired mainly engineers to date, we are now growing across all areas of the business and we are looking to hire someone that is able to take complete ownership of our marketing - from strategy to execution, as the first hire.
+
+This role will be focused on increasing user acquisition, and community growth around our open source project. It's key to PostHog's success and will report directly to the CEO.
+
+The primary goal of this role is to increase the number of new users into our open source project. We currently get around 120 deployments a week organically, but we want to accelerate this growth. Secondarily, we want to increase our community size (it currently exists in our Slack user group, GitHub, our mailing list and Twitter). 
+
+We hire the best to tell us what to do. We'd expect to give you a healthy budget to achieve the above, and we're well-financed. A great candidate will be able to steer us as to what type of spend is appropriate, and will be happy doing the work whilst steering the ship. 
+
+## What you'll bring:
+* Ability to understand a developer's mindset - you might have been a developer in the past or you might have collaborated very closely with engineering teams
+* Strong Marketing experience across digital (content, social) and brand
+* Strong background in writing content for a developer audience
+* Experience in early-stage, high-growth companies, or you might have been a founder yourself
+* Some paid experience a plus
+* Ideally, you have experience in marketing in developer-focused companies
+* You've led marketing in some capacity before. You need not to have managed a huge team, but we do need someone that has demonstrably taken initiative before with a proven track record of success. The ideal candidate is comfortable leading, but wants to be the first person in at a hyper growth company.
```

Comment:
I think we should change this to be "you don't need to have managed a team before" or remove entirely as I think we'd exclude some very strong hands-on candidates.

Reply:
Fixed